examples/convert_ieeg_to_bids.py

```diff
@@ -331,3 +335,133 @@
+
+# define a transform to MNI152NLin2009bAsym (fake it)
+# MNE-Python doesn't recognize MNI152NLin2009bAsym, so we have to use
+# the unknown coordinate frame
+head_template_t = np.array([[1.0, 0.1, 0.2, 10.1],
+                            [-0.1, 1.0, 0.1, -20.3],
+                            [0.2, -0.1, 1.0, -30.7],
+                            [0.0, 0.0, 0.0, 1.0]])
+head_template_trans = mne.transforms.Transform(
+    fro='head', to='unknown', trans=head_template_t)
+montage.apply_trans(head_template_trans)
+
+# write to BIDS, with the montage in fsaverage coordinates
+write_raw_bids(raw, bids_path, anonymize=dict(daysback=40000),
+               montage=montage, overwrite=True)
+
+# %%
+# Now, let's see how we would work with the data in MNE. As shown below, the
+# montage has the same coordinates as when it was written, but the coordinate
+# frame is unknown.
+
+# read back in the raw data
+raw = read_raw_bids(bids_path=bids_path)
+
+# get the montage
+montage2 = raw.get_montage()
+print('Montage set to: ' + montage2.get_positions()['coord_frame'])
+
+# check that we can recover the coordinates
+print('Recovered coordinate: {recovered}\n'
+      'Saved coordinate:     {saved}'.format(
+          recovered=montage2.dig[0]['r'],
+          saved=montage.dig[0]['r']))
+
+# %%
+# To work with this data in the template coordinate frame, all the same steps
+# can be followed in :ref:`tut-working-with-seeg` and
+# :ref:`tut-working-with-ecog` after the montage is transformed to surface
+# RAS coordinates for the template MRI (if it isn't there already).
+#
+# First we'll need the ``subjects_dir`` where the recon-all for the template
+# brain is stored.
+#
+# .. code-block:: python
+#
+#    subjects_dir = op.join(misc_path, 'subjects')  # for example
+#    # add some plotting keyword arguments
+#    brain_kwargs = dict(cortex='low_contrast', alpha=0.2, background='white')
+#
+# If the montage is already in surface RAS for the template MRI, we can use:
+#
+# .. code-block:: python
+#
+#      # identity transform since 'unknown' is already 'mri' == surface RAS
+#      # for the template brain MRI
+#      trans = mne.transforms.Transform(
+#          fro='unknown', to='mri', trans=np.eye(4))
+#      brain = mne.viz.Brain('sample_seeg', subjects_dir=subjects_dir,
+#          **brain_kwargs)
+#      brain.add_sensors(raw.info, trans=trans)
+#
+# If the montage was in voxel coordinates, we'll first have to transform
+# to surface RAS:
+#
+# .. code-block:: python
+#
+#      import nibabel as nib
+#      template_T1 = nib.load(op.join(subjects_dir, 'MNI152NLin2009bAsym',
+#                                     'mri', 'T1.mgz'))
+#      trans = mne.transforms.Transform(  # use vox to surface RAS transform
+#          fro='unknown', to='mri', trans=template_T1.header.get_vox2ras_tkr())
+#      brain = mne.viz.Brain(
+#          'sample_seeg', subjects_dir=subjects_dir, **brain_kwargs)
+#      brain.add_sensors(raw.info, trans=trans)
```

Comment:
Can you move the closing parens to a new line please? I think that's what we're mostly using now for new code in MNE-Python and also in MNE-BIDS.

e.g.,
```python
Foo(
    bar=baz
)
```